### ISSUE TITLE
[BUG]showing nodes' movement in core in sdt3d went wrong

### ISSUE THREAD

**Issue by Revan-github** [coreemu/core#545]
**Describe the bug**
when I run a core sample with moving nodes, the nodes in sdt3d didn't move at all but the link is changed according to the links in core.
However, when I move the nodes by hand, the node in sdt3d moved as well.
Here is the error message on sdt3d command line
I'd like to know if it is caused by the code in core or my environment. 
And is there any possible way to fix it.
**Here is a screen shot for the error message in sdt3d: **, it seems that it went wrong when core create layers and attach nodes to layers.

**To Reproduce**
1. start core-daemon by sudo
2. start core-gui or core-pygui
3. open example1.xml/imn and set sdt3d view to true
4. start sdt3d and set tcp listen to port 50000
5. start the emulation
6. **Error: **the nodes in sdt3d didn't move when the nodes in core was moving but the wireless links' status is right. 

**Expected behavior**
the nodes and links in sdt3d can move or change according to these in core

**Screenshots**
![QQ图片20201225175329](https://user-images.githubusercontent.com/45191163/103131046-26a38e80-46da-11eb-9394-5c1260d0b687.jpg)

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - CORE Version : 7.3.0
 - EMANE Version: 1.2.5
 - sdt3d version: 2.3.0 or 2.1

**Additional context**

**Comment by bharnden** [comment]
I would have SDT started 1st at step 1, that may be your issue.

**Comment by Revan-github** [comment]
I solved this problem. 
I found out that it is the configuration of wlan10 node is wrong. 
I change it to EMANE(I tested ieee802.11 and the one before it) model and change the length of subnet from 32 to  24.
Finally, it worked.
Still, I'm not so clearly about why it fixed the problem. 
Is it because the wrong wlan10's configuration made all the nodes under wlan10 can not be loaded in sdt3d properly?

Here is another problem, I find that the default wlan configuration defines the ipv4 subnet's length as 32 and the ipv6 subnet's length is 128, which is not right , I often need to fix it by hand. 

Should I start a new issue to report this or just edit this issue?
